### DISCUSSION TITLE
Not able to access nomination api when trying to hit it via apache

### DISCUSSION THREAD

**Discussion by Shadow-shadown** [osm-search/Nominatim#3278] [General]
I have Nominatim setup in ubuntu22 when I run the nominatim search --query New York I'm getting the output but when I try to access it through apache in I'm getting 

ERROR: Database connection failed 

How do I go about this ? 
Any help would be appreciated 
Thank you

**Comment by mtmail** [comment]
Can you share which version of Nominatim and which installation instructions you followed? I saw you asked questions in the nominatim-docker as well so it's important to know which installation you used. What command did you run which returned "ERROR: Database connection failed". Have you done any changes to the Apache configuration files?

**Reply by Shadow-shadown** [reply]
I'm trying to setup a geocoding service so I'm trying both mediagis as well as manual setup of nominatim
So while setting it up manually 
I'm getting this error 

•![image](https://github.com/osm-search/Nominatim/assets/107956883/e6755a4e-11bf-4bc2-8482-efae1f112538)

My postgres databases 

![image](https://github.com/osm-search/Nominatim/assets/107956883/4790fd67-d10a-41ce-91d4-337ad62a62dc)

Apache config inside conf-available/nominatim.conf

**Reply by mtmail** [reply]
Can you share which version of Nominatim and which installation instructions you followed? What command did you run which returned "ERROR: Database connection failed". Have you done any changes to the Apache configuration files compared to the installation instructions?

**Reply by Shadow-shadown** [reply]
Nominatim version 4.3.2 
For Ubuntu 22.04 with the installation instructions on the official nominating website with no changes to the apache config file 
But I'm still getting that issue

**Reply by mtmail** [reply]
What command did you run which returned "ERROR: Database connection failed"?

**Reply by Shadow-shadown** [reply]
After I setup I went to the apache endpoint and tried status.php to make sure if it was working 
When trying it showed this message in the browser 
But when I run the Nominatim command 
 nominatim search --query London in my Ubuntu server it gives the result but not able to provide it through apache

**Reply by mtmail** [reply]
Your screenshot shows `Alias /nominatim /srv/nominatim/build/website`. The second path needs to point to the website directory in the project path, not the build path. Try `Alias /nominatim /user/ubuntu/nominatim-project/website`. I don't know which tutorial you followed or what paths you have on your system, I'm looking at https://nominatim.org/release-docs/latest/appendix/Install-on-Ubuntu-22/
